extensions/amp-list/0.1/amp-list.js

```diff
@@ -947,86 +1059,44 @@ export class AmpList extends AMP.BaseElement {
   }
 
   /**
-   * @return {!Element}
+   * @param {boolean} opt_refresh
+   * @return {!Promise<(!Array<JsonObject>|!JsonObject)>}
    * @private
    */
-  getLoadMoreFailedElement_() {
-    if (!this.loadMoreFailedElement_) {
-      this.loadMoreFailedElement_ = childElementByAttr(
-          this.element, 'load-more-failed');
-
-      if (!this.loadMoreFailedElement_) {
-        this.loadMoreFailedElement_ = htmlFor(this.win.document)`
-          <amp-list-load-more load-more-failed class="i-amphtml-default-ui">
-            <div class="i-amphtml-list-load-more-message">
-              Unable to Load More
-            </div>
-            <button load-more-clickable
-              class="i-amphtml-list-load-more-button
-                     i-amphtml-list-load-more-button-has-icon
-                     i-amphtml-list-load-more-button-small"
-            >
-              <div class="i-amphtml-list-load-more-icon"></div>
-              <label>Retry</label>
-            </button>
-          </amp-list-load-more>
-        `;
-      }
-    }
-    return this.loadMoreFailedElement_;
+  fetch_(opt_refresh = false) {
+    return batchFetchJsonFor(
+        this.getAmpDoc(), this.element, '.', this.getPolicy_(), opt_refresh);
   }
 
   /**
+   * TODO: debounce this.
```

Comment:
I think this is a regression caused by the refactor of position observer to use `viewport.onScroll`. It currently triggers way too often. Intending to debug and fix this in a separate PR.